meetings/2024-06-12.md

```diff
@@ -28,11 +35,123 @@
+* HZ: Jeff, Is it a strong objection or preference?
+* JZ: I think Sarven and Virginia should be present. Some concerns need to be addressed.
+* HZ: Do you agree that it is orthogonal to your issue?
+* JZ: No I don't agree
+* HZ: From architecture point of view I agree with Pavlik. If you store WebID document on the Solid Pod, if pod is not available the webid document is also not available. But I'm not going to die on that hill.
+* JZ: There are 50k pods on solidcommunity.net those people and those pods I'm concern about. It is not orthogonal. There is an existing situation of knowledgable people having WebIDs out of the Solid server. At minimum Sarven and Virginia need to be present.
+* HZ: I agree with all of you, let's take it offline.
+* Ted: One of the challenges I had in PR9, the structure of the document needs to be more consumable and actionable. There is a blur between weaknesses and how to address them. The countermeasures are at the same level as the weaknesses. One is a theft of the token, there is another with theft of the credentials. There should be weakenss, attack and countermeasure that prevents it. And considerationsa about that weakenss. 
+* eP: It will be easier to structure based on more than one weakness.
+* HZ: I see more thought about infrastructure and not only focus on p2p
+
+
+---
+* VB (following minutes but cannot join): Voting does not necessarily help with consensus. The issues with the PR are not objecting to the security consideration per se, but more seeking to add considerations with the approach. I am not opposed to hosting webid outside solid storage (obviously), but this PR seems to advocate for *not* hosting it on Solid storage, which is a strong message. I think we should look into how to securely allow to host. Happy to discuss next week / in another meeting. 
+* VB: See: https://github.com/solid/security-considerations/pull/9#issuecomment-2155333752 on how to move forward.
+
+
+Meeting ends
+
+### N3 eligibility for being a normative reference in W3C WG deliverable
+
+
+URL: https://solidproject.org/TR/protocol#normative-references
+URL: https://w3c.github.io/N3/spec/
+
+> https://github.com/solid/solid-wg-charter/issues/37#issuecomment-1580600098
+> PAC: I think I remember @timbl talking about this not so long (during a Solid CG meeting??): as I remember, he was explaining that W3C RECs can depend on things that are one step behind in terms of maturity, precisely to make the co-evolution of standards more agile. I could not find anything about this in the current W3C process, though... But @csarven's example above shows there are precedents.
+
+* eP: We should clarify ASAP if N3 can be used as normative reference in any of WG deliverables. If it has to be one step behind in maturity we need to confirm if it is planned to be picked up by a WG in a near future and plan accordingly.
```

Comment:
```suggestion
* eP: We should clarify ASAP if N3 can be used as normative reference in Solid WG deliverables. If it can be no more than one step behind in maturity, we need to confirm whether it is planned to be picked up by a WG in the near future and plan accordingly.
```